src/components/Images/Images.js

```diff
@@ -162,19 +148,7 @@ export const CloudinaryImage = ({
         imageSettings
       )
 
-      const urlWithChainedLqipTransformation = createCloudinaryLegacyURL(
```

Comment:
After discussing with @t1mwillis , we believe that not using LQIP at all reduces the size of the img resource requested substantially and is a worthy tradeoff.